src/lsps1/event.rs

```diff
@@ -10,64 +19,104 @@ use bitcoin::secp256k1::PublicKey;
 /// An event which an LSPS1 client should take some action in response to.
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub enum LSPS1ClientEvent {
-	/// TODO
+	/// Information from the LSP about their supported protocol options.
+	///
+	/// You must check whether LSP supports the params the client wants and then call
+	/// [`LSPS1ClientHandler::place_order`] to place an order.
+	///
+	/// [`LSPS1ClientHandler::place_order`] : crate::lsps1::client::LSPS1ClientHandler::place_order
 	GetInfoResponse {
-		/// TODO
+		/// This is a randomly generated identifier used to track the channel state.
```

Comment:
nit: Leave one line whitespace after initial description line.
```suggestion
		/// This is a randomly generated identifier used to track the channel state.
		///
```